### PR TITLE
Update kubemark/config-default.sh

### PR DESCRIPTION
**Pull request by gmarek**
Update kubemark/config-default.sh

### CODE REVIEW

File: cluster/kubemark/config-default.sh [hunk 1]
```diff
@@ -39,6 +39,20 @@ MASTER_TAG="kubemark-master"
 MASTER_IP_RANGE="${MASTER_IP_RANGE:-10.246.0.0/24}"
 CLUSTER_IP_RANGE="${CLUSTER_IP_RANGE:-10.244.0.0/16}"
 RUNTIME_CONFIG="${KUBE_RUNTIME_CONFIG:-}"
+TERMINATED_POD_GC_THRESHOLD=${TERMINATED_POD_GC_THRESHOLD:-100}
+
+# Optional: enable v1beta1 related features
+ENABLE_DEPLOYMENTS="${KUBE_ENABLE_DEPLOYMENTS:-true}"
+ENABLE_DAEMONSETS="${KUBE_ENABLE_DAEMONSETS:-true}"
+
+TEST_CLUSTER_LOG_LEVEL="${TEST_CLUSTER_LOG_LEVEL:---v=4}"
```
Comment (wojtek-t) [inline cluster/kubemark/config-default.sh]: I think we want the default to be the same as for scalability tests:
https://github.com/kubernetes/kubernetes/blob/master/hack/jenkins/e2e.sh#L419
which is "--v=2"

File: cluster/kubemark/config-default.sh [hunk 2]
```diff
@@ -39,6 +39,20 @@ MASTER_TAG="kubemark-master"
...
+TEST_CLUSTER_RESYNC_PERIOD="${TEST_CLUSTER_RESYNC_PERIOD:---min-resync-period=3m}"
```
Comment (wojtek-t) [inline cluster/kubemark/config-default.sh]: Same here: ""--min-resync-period=12h":
https://github.com/kubernetes/kubernetes/blob/master/hack/jenkins/e2e.sh#L421

File: cluster/kubemark/config-default.sh (hunk 1 above)
Comment (gmarek) [inline cluster/kubemark/config-default.sh]: Done.

File: cluster/kubemark/config-default.sh (hunk 2 above)
Comment (gmarek) [inline cluster/kubemark/config-default.sh]: Done.

**Comment by wojtek-t** [comment]
LGMT

**Pull request merged**